### PR TITLE
[TFTRT - Dynamic Shape Phase 3] Add Dynamic Shape Testing for ConvertArgMinMax

### PR DESCRIPTION
**Pull request by DEKHTIARJonathan**
@bixia1 @tfeher for review

Feature Tracker: #45481

### CODE REVIEW

**Review by bixia1 — COMMENTED** [review]
Looks good to me. Thanks for working on this!
The test passes with TRT7 but fails with TRT5 though.

**Comment by gbaned** [comment]
@DEKHTIARJonathan  Can you please check @bixia1's comments and keep us posted ? Thanks!

**Comment by DEKHTIARJonathan** [comment]
@bixia1 I found the issue:

```
DefaultLogger my_arg-topk: implementation limit for 4D and higher tensors -- reduceAxes must specify one of last three dimensions
```

I pushed the fix and now it passes on TRT5 & TRT7

File: tensorflow/compiler/tf2tensorrt/convert/utils.cc [hunk 1]
```diff
@@ -186,6 +186,25 @@ Status TrtDimsToTensorShape(const nvinfer1::Dims trt_dims,
   return Status::OK();
 }
 
+string TrtDimsToTensorShapeString(const nvinfer1::Dims trt_dims,
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/utils.cc:189]: How about using absl::StrCat and absl::StrJoin like here https://github.com/tensorflow/tensorflow/blob/0c79db02d59e7799b3dcf3aff31e0bad71bc809d/tensorflow/compiler/tf2tensorrt/convert/utils.cc#L122-L127
?

Comment (DEKHTIARJonathan) [reply tensorflow/compiler/tf2tensorrt/convert/utils.cc]: Oh right. I didn't see that function, my bad.

It's fixed ;)

**Comment by DEKHTIARJonathan** [comment]
@bixia1 we should be good to go ;)

**Review by bixia1 — APPROVED** [review]

**Pull request merged**